library/edihistory/test_edih_sftp_files.php

```diff
@@ -378,3 +378,4 @@ function sftp_status($msg, $val)
     fwrite(($exitcd ? STDERR : STDOUT), xl($exitmsgs[$exitcd]).PHP_EOL);
```

Comment:
This script makes me a bit nervous since there is no authentication happening here and stuff is happening outside the functions. Recommend placing the below at line 25:
```
// comment out below exit when need to use this script
exit;
```